modules/con_quay_ha_prereq.adoc

```diff
@@ -46,33 +46,28 @@ Each system should have the following attributes:
 
 //* **Red Hat Enterprise Linux (RHEL)**: Obtain the latest Red Hat Enterprise Linux server media from the link:https://access.redhat.com/downloads/content/69/ver=/rhel---7/7.5/x86_64/product-software[Downloads page] and follow instructions from the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html-single/installation_guide/index[Red Hat Enterprise Linux 7 Installation Guide] to install RHEL on each system.
 //** **Valid Red Hat Subscription**: Obtain Red Hat Enterprise Linux server subscriptions and apply one to each system.
-* **Red Hat Enterprise Linux (RHEL)**: Obtain the latest Red Hat Enterprise Linux 8 server media from the link:https://access.redhat.com/downloads/content/479/ver=/rhel---8/8.3/x86_64/product-software[Downloads page] and follow the installation instructions available in the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/[Product Documentation for Red Hat Enterprise Linux 8].
+* **Red Hat Enterprise Linux (RHEL)** 8: Obtain the latest Red Hat Enterprise Linux 8 server media from the link:https://access.redhat.com/downloads/content/479/ver=/rhel---8/8.3/x86_64/product-software[Downloads page] and follow the installation instructions available in the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/[Product Documentation for Red Hat Enterprise Linux 8].
 ** **Valid Red Hat Subscription**: Configure a valid Red Hat Enterprise Linux 8 server subscription.
 
 ** **CPUs**: Two or more virtual CPUs
 ** **RAM**: 4GB for each A and B system; 8GB for each C system
 ** **Disk space**:  About 20GB of disk space for each A and B system (10GB for the operating system and 10GB for docker storage).  At least 30GB of disk space for C systems (or more depending on required container storage).
 
-[NOTE]
-====
```

Comment:
Has RHEL 7 been deprecated? If not, I'd leave the note here and just change 3.6->3.7